apps/web/src/ee/clerk/pages/ManageAccountPage.tsx

```diff
@@ -0,0 +1,149 @@
+import './ManageAccountPage.css';
+import { OrganizationProfile, UserProfile } from '@clerk/clerk-react';
+import { useColorScheme } from '@novu/design-system';
+import { Modal, Tabs } from '@mantine/core';
```

Comment:
This should be Mantine v5, right? Can we use Mantine 7?

Reply:
yes, we use v5 within the dashboard project. We use v7 in novui, but I quickly came to realize that there would need to be some adjustments in this package to make it work the way we want and it took too much time off my plate. We can create a follow up task for this in the backlog.